components/chart/index.tsx

```diff
@@ -2,7 +2,7 @@ import { Card, CardContent, Typography } from '@mui/joy';
 import BarChart from './bar-chart';
 import LineChart from './line-chart';
 import TableChart from './table-chart';
-import { ChartData } from '@/types/chart';
+import { ChartData } from '@/types/chat';
```

Comment:
`chat` and `chart ` are not same. and there will be more types about chart later .